src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java

```diff
@@ -155,6 +156,7 @@
 public class BigQueryEventConsumer implements EventConsumer {
   private static final Logger LOG = LoggerFactory.getLogger(BigQueryEventConsumer.class);
   private static final int MAX_LENGTH = 1024;
+  private static final int MAX_CLUSTERING_COLUMNS = 4;
```

Comment:
can we make this overridable through a runtime argument? I'm thinking it would be better in case BQ increases the max cluster number. then we don't need a code change to take advantage, we could tell users to set a preference, and then restart their replicators.

Reply:
Sure, this is a good call out. We can definitely do that. will update.

Reply:
Also added unit test in following commit.